2018-November-27/writingSample/rangel.tex

```diff
@@ -0,0 +1,155 @@
+time of the TMS pulse, their neurotransmitter release will be shunted,
+and the MEP will have decreased amplitude (Cortico-Spinal Inhibition,
+Figure 2.). TMS remains as one of the primary tools for the measurement
+of inhibition at the cortical level, with applications for pharmacology,
+physical therapy and cognitive studies.
+
+However, TMS presents both experimental issues and unexplained phenomena
+that must be taken into account during research. TMS coil direction
+alone has been hypothesized to preferentially activate different
+populations of M1 interneurons, resulting in different effects on the
+MEP (Figure 3.). Alternate coils (H-Coils), have been devised to reduce
+any pulse-to-pulse variability in position due to experimenter error,
+but they are so far largely restricted to clinical settings.
+Additionally, stands used to hold the TMS coil to the head of the
+participant can also eliminate placement variability, but require
+immobility on the part of the participant, which can limit the
+flexibility of behavioral tasks. Ultimately, even when the TMS coil
+position is inert, MEP amplitudes are known to be vary pulse-to-pulse
+and can be suppressed by the TMS pulses themselves over the course of a
+longer experiment. TMS pulses also produce loud sounds and haptic
+sensations on the skull, which can become distracting and/or startling
+at higher intensities, and activate other, non-target, muscles. Finally,
+patients with epilepsy or traumatic brain injury are often excluded from
+TMS studies as a precaution, which eliminates the prospect of studying
+inhibition in such a unique population. Human studies looking at
+Cortico-Spinal inhibition with TMS must be designed such that they
+accommodate these physiological drawbacks. While TMS still remains a
+wonderful method for many task paradigms, it becomes particularly
+troublesome in tasks where small effect sizes may be susceptible to MEP
+variability, the patient population cannot use TMS, or where
+loud/jarring TMS pulses could alter the inhibitive responses.
+
+The challenges of using TMS to measure inhibition have prompted the
+investigation into more universal methods. Some recent research has
+focused on correlating Electroencephalography (EEG) signatures
+(fronto-central P3), with decreased force output and/or reduced muscular
+activity. For example, in tasks where inhibition promoting (unexpected)
+stimuli are presented, participants can be instructed to hold a force
+meter at a constant level (Figure 4.). Previous findings have shown that
+both EEG signatures and force output of the hand show distinct
+modulations following unexpected stimuli, which are consistent with an
+inhibitory process (Figure 5.). Further investigations are currently
+underway to evaluate if the EEG/Force method of measuring Cortico-Spinal
+inhibition can viably substitute for TMS, during cognitive tasks. If so,
+many of the previously unavailable task paradigms and patient
```

Comment:
I wonder what task paradigms, are they related to measuring cortical inhibition?